Dockerfile

```diff
@@ -27,7 +27,7 @@ ENV GIT_BRANCH=${GIT_BRANCH}
 RUN apt-get update && \
     apt-get upgrade -y && \
     apt-get autoremove -y && \
-    apt-get install -y make python g++ curl && \
+    apt-get install -y make python g++ curl default-jre && \
```

Comment:
How come we're adding a java-runtime-environment to a node-based container?

Reply:
Its because i needed to use wiremock and the dev environment uses docker. Im looking to see if we can still use wiremock, but start it outside of the container. Also discussing this with Alex

Reply:
reverted this change, now using docker

Reply:
Great stuff, resolving